packages/manager/apps/ips/src/pages/onboarding/index.tsx

```diff
@@ -127,32 +170,24 @@ export default function Onboarding() {
                 {t('advantagesTitle')}
               </OdsText>
               <div className="text-left mb-8">
-                <ol className="list-decimal pl-6 space-y-4">
-                  <li>
-                    <strong>{t('advantage1Title')}:</strong>{' '}
-                    {t('advantage1Description')}
-                  </li>
-                  <li>
-                    <strong>{t('advantage2Title')}:</strong>{' '}
-                    {t('advantage2Description')}
-                  </li>
-                  <li>
-                    <strong>{t('advantage3Title')}:</strong>{' '}
-                    {t('advantage3Description')}
-                  </li>
-                  <li>
-                    <strong>{t('advantage4Title')}:</strong>{' '}
-                    {t('advantage4Description')}
-                  </li>
+                <ul className="list-disc pl-6 space-y-4">
+                  {IP_OPTIONS_ADVANTAGES}
```

Comment:
Prefer creating a separate component and use jsx instead
```
<IpOptionsAdvantages />
```

Reply:
done